### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by min-star** [continue-revolution/sd-webui-animatediff#225]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### Have you read FAQ on README?

- [X] I have updated WebUI and this extension to the latest version

### What happened?

i have update webui to 1.6 and controlnet 1.141,but enabled animatediff,error breaked

### Steps to reproduce the problem

![截图 2023-10-16 22-22-13](https://github.com/continue-revolution/sd-webui-animatediff/assets/59134868/bcd7ffe5-2945-4c88-a827-ed70233522df)

### What should have happened?

![截图 2023-10-16 22-25-56](https://github.com/continue-revolution/sd-webui-animatediff/assets/59134868/b0424d5c-e4c3-4132-851b-d3cdf112af54)

### Commit where the problem happens

AttributeError: 'AnimateDiffProcess' object has no attribute 'video_source'

### Command Line Arguments

```Shell
python3 launch.py --no-gradio-queue --listen --port 12121 --xformers --share --skip-prepare-environment --enable-insecure-extension-access --api
```

### Console logs

```Shell
2023-10-16 14:25:33,871 - ControlNet - INFO - ControlNet v1.1.410
ControlNet preprocessor location: /workspace/extensions/sd-webui-controlnet/annotator/downloads
2023-10-16 14:25:33,910 - ControlNet - INFO - ControlNet v1.1.410
Loading weights [8a952cafe9] from /workspace/models/Stable-diffusion/anyloraCheckpoint_bakedvaeFtmseFp16NOT.safetensors
Running on local URL:  http://0.0.0.0:12121
Creating model from config: /workspace/configs/v1-inference.yaml
Applying attention optimization: xformers... done.
Model loaded in 1.3s (load weights from disk: 0.3s, create model: 0.3s, apply weights to model: 0.6s).
2023-10-16 14:27:00,944 - AnimateDiff - INFO - AnimateDiff process start.
2023-10-16 14:27:00,944 - AnimateDiff - INFO - You are using tested mm mm_sd_15.ckpt.
2023-10-16 14:27:00,944 - AnimateDiff - INFO - Loading motion module mm_sd_v15.ckpt from /workspace/extensions/sd-webui-animatediff/model/mm_sd_v15.ckpt
2023-10-16 14:27:02,817 - AnimateDiff - WARNING - Missing keys <All keys matched successfully>
2023-10-16 14:27:02,984 - AnimateDiff - INFO - Hacking GroupNorm32 forward function.
2023-10-16 14:27:02,984 - AnimateDiff - INFO - Injecting motion module mm_sd_v15.ckpt into SD1.5 UNet input blocks.
2023-10-16 14:27:02,984 - AnimateDiff - INFO - Injecting motion module mm_sd_v15.ckpt into SD1.5 UNet output blocks.
2023-10-16 14:27:02,984 - AnimateDiff - INFO - Setting DDIM alpha.
2023-10-16 14:27:03,088 - AnimateDiff - INFO - Injection finished.
2023-10-16 14:27:03,089 - AnimateDiff - INFO - Hacking CFGDenoiser forward function.
2023-10-16 14:27:03,089 - AnimateDiff - INFO - Hacking ControlNet.
*** Error completing request
*** Arguments: ('task(8u3dvhn285ai7pn)', '', '', [], 20, 'DPM++ 2M Karras', 1, 1, 7, 512, 512, False, 0.7, 2, 'Latent', 0, 0, 0, 'Use same checkpoint', 'Use same sampler', '', '', [], <gradio.routes.Request object at 0x7f8dd6b7c0d0>, 0, False, '', 0.8, -1, False, -1, 0, 0, 0, <scripts.animatediff_ui.AnimateDiffProcess object at 0x7f8dfc9f0070>, <scripts.controlnet_ui.controlnet_ui_group.UiControlNetUnit object at 0x7f8dfc9f28f0>, <scripts.controlnet_ui.controlnet_ui_group.UiControlNetUnit object at 0x7f8dfc9f20e0>, <scripts.controlnet_ui.controlnet_ui_group.UiControlNetUnit object at 0x7f8dfc9f2080>, False, False, 'positive', 'comma', 0, False, False, '', 1, '', [], 0, '', [], 0, '', [], True, False, False, False, 0, False, None, None, False, None, None, False, None, None, False, 50) {}
    Traceback (most recent call last):
      File "/workspace/modules/call_queue.py", line 57, in f
        res = list(func(*args, **kwargs))
      File "/workspace/modules/call_queue.py", line 36, in f
        res = func(*args, **kwargs)
      File "/workspace/modules/txt2img.py", line 55, in txt2img
        processed = processing.process_images(p)
      File "/workspace/modules/processing.py", line 732, in process_images
        res = process_images_inner(p)
      File "/workspace/extensions/sd-webui-animatediff/scripts/animatediff_cn.py", line 67, in hacked_processing_process_images_hijack
        global_input_frames = get_input_frames()
      File "/workspace/extensions/sd-webui-animatediff/scripts/animatediff_cn.py", line 39, in get_input_frames
        if params.video_source is not None and params.video_source != '':
    AttributeError: 'AnimateDiffProcess' object has no attribute 'video_source'
```

**Comment by ArtemKo7v** [comment]
Same issue with ControlNet enabled

**Comment by zappityzap** [comment]
I reproduced this error with a video loaded in the video_source input and ControlNet enabled. Haven't been able to figure out why yet.

**Comment by continue-revolution** [comment]
The reason is simply a miss of `copy()`

**Comment by 4lt3r3go** [comment]
> The reason is simply a miss of `copy()`

OK were should i paste this code?

**Comment by continue-revolution** [comment]
@4lt3r3go This is completely my bad and I've already fixed it in #227 

It should be fine after you update this extension,

**Comment by 4lt3r3go** [comment]
> @4lt3r3go This is completely my bad and I've already fixed it in #227
> 
> It should be fine after you update this extension,

THANK YOU!!! you saved my day
